morevna-ep4.msrt

```diff
@@ -241,7 +241,7 @@
 49
 00:05:36,880 --> 00:05:37,720
 [rus] *Не мешайтесь!*
-[eng] *Stay away!*
+[eng] *Stay clear!*
```

Comment:
If someone was saying this in reference to applying a defibrillator, then "Stay clear!" or "Stand clear!" are the two best options. Based on the sound effect it seems like this is the case. If not, I'd have to think about what the best phrasing would be. "Stay away!" implies some kind of danger in the area like "Stay away, he may be contagious!". If Ivan is simply getting in the way, then it would be "Go away", "Get out of the way", etc. as you used earlier, since he obviously needs to leave, not stay where he is.

Reply:
Yes, the meaning is that Ivan is simply getting in the way. So "Get out of the way" fits best. ^__^

Reply:
I changed this to "Get out of the way" and modified the earlier "Get out of my way" line to something else so that it doesn't seem repetitive.